scripts/stac/imagery/tests/conftest.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+from typing import Generator
+
+import pytest
+
+from scripts.stac.imagery.metadata_constants import CollectionMetadata
+
+
+@pytest.fixture(autouse=True)
+def fake_collection_metadata() -> Generator[CollectionMetadata, None, None]:
```

Comment:
I believe the return type can be simplified to `Iterator[CollectionMetadata]` in general, when the second and third `Generator` types are `None`.